modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -7,34 +7,64 @@
 
 Uninstalling the web terminal is a two-step process:
 
-. Delete the components and custom resources (CRs) that were added when you installed the Operator.
-. Uninstall the Web Terminal Operator.
+. Uninstall the Web Terminal Operator and related custom resources (CRs) that were added when you installed the Operator.
+. Uninstall the DevWorkspace Operator and its related custom resources that were added as a dependency of the Web Terminal Operator.
 
 Uninstalling the Web Terminal Operator does not remove any of its custom resource definitions (CRDs) or managed resources that are created when the Operator is installed. These components must be manually uninstalled for security purposes. Removing these components also allows you to save cluster resources by ensuring that terminals do not idle when the Operator is uninstalled.
 
 .Prerequisites
 * Access to an {product-title} cluster using an account with `cluster-admin` permissions.
 
-== Deleting the web terminal components and custom resources
+== Removing the Web Terminal Operator and the custom resources that support it
 
-Use the CLI to delete the CRs that are created during installation of the  Web Terminal Operator.
+Use the CLI to delete any existing web terminals and CRs that were created during the installation of the Web Terminal Operator.
+
+[NOTE]
+====
+Prior to {product-title} 4.8, the Web Terminal Operator used different CRDs to provide Web Terminal capabilities. To uninstall versions 1.2.1 and below of the Web Terminal Operator, refer to the documentation for {product-title} 4.7.
+====
 
 .Procedure
-. Run the following commands to ensure that all `DevWorkspace` CRs are removed along with their related Kubernetes objects, such as deployments.
+. Run the following commands to ensure that all `DevWorkspace` CRs created for Web Terminals are removed.
 +
 [source,terminal]
 ----
-$ oc delete devworkspaces.workspace.devfile.io --all-namespaces --all --wait
+$ oc delete devworkspaces.workspace.devfile.io --all-namespaces \
+    --selector 'console.openshift.io/terminal=true' --wait
 ----
 +
 [source,terminal]
 ----
-$ oc delete workspaceroutings.controller.devfile.io --all-namespaces --all --wait
+$ oc delete devworkspacetemplates.workspace.devfile.io --all-namespaces \
+    --selector 'console.openshift.io/terminal=true' --wait
 ----
 +
+. Uninstall the Web Terminal Operator via the web console:
```

Comment:
Maybe it should be the first step and then clean up CRs, to avoid their creation before admin got to the second step.

Reply:
Good point, fixed!